pkgs/by-name/ar/argc/package.nix

```
@@ -37,6 +38,11 @@ rustPlatform.buildRustPackage rec {
 
   disallowedReferences = lib.optional (!canExecuteHost) buildPackages.argc;
 
+  env = {
+    LANG = "en_US.UTF-8";
+    LOCALE_ARCHIVE = "${glibcLocales}/lib/locale/locale-archive";
```

Ah, I got it!

https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/all-packages.nix#L20736-L20744


```suggestion
    LOCALE_ARCHIVE = lib.optionalString (glibcLocale != null) "${glibcLocales}/lib/locale/locale-archive";
```

---

I was scratching my head about what do with this one :)

Great work! I've included the suggestion in 4d248dad5e5391126c3730003b469c022e041713.